Cargo.toml

```diff
@@ -28,17 +28,17 @@ signal-hook = "0.1.10"
 strip-ansi-escapes = "0.1.0"
 structopt = "0.3"
 # termion = { git = "https://gitlab.com/TheLostLambda/termion.git", version = "1.6.0", features = ["serde"] }
-termion = { package = "termion_temporary_zellij_fork" , version = "1.6.0", features = ["serde"]}
+termion = "1.5.0"
```

Comment:
Woohoo! Why do we need the comment still?

Reply:
Whoops! We definitely don't! I just missed that! Gone in the next commit :)